lib/Transforms/Tapir/OpenMPABI.cpp

```diff
@@ -803,6 +803,8 @@ void llvm::OpenMPABI::postProcessFunction(Function &F) {
       OpenMPRuntimeFunction::OMPRTL__kmpc_fork_call, F.getParent());
   // Replace the old call with __kmpc_fork_call
   auto *ForkCall = emitRuntimeCall(ForkRTFn, OMPRegionFnArgs, "", b);
+  assert(ForkCall != 0); // play it safe -- something better to do here?
+  
```

Comment:
I'm being nit-picky, but for matching code style, I think we would write `assert(ForkCall && "Some helpful error message.");` or `assert(ForkCall != nullptr && "Some helpful error message.");`